src/test/scala/org/constellation/consensus/RoundTest.scala

```diff
@@ -337,10 +339,51 @@ class RoundTest
   }
 
   test("it should accept majority checkpoint block") {
+    round ! SelectedUnionBlock(
+        roundId,
+        facilitatorId1,
+        cb1
+    )
+    round ! SelectedUnionBlock(
+        roundId,
+      facilitatorId2,
+        cb1
+    )
+
+    round ! SelectedUnionBlock(
+        roundId,
+        facilitatorId3,
+        cb2
+    )
+    round.underlyingActor.passToParentActor(StopBlockCreationRound(RoundId("round1"), Some(cb1))) was called
+  }
+
+  test("it should return none when accept majority checkpoint block failed") {
+    dao.threadSafeSnapshotService.accept(*) shouldAnswer ((a: CheckpointCache) => Failure(TipConflictException(cb1)))
+    round ! SelectedUnionBlock(
+      roundId,
+      facilitatorId1,
+      cb1
+    )
+    round ! SelectedUnionBlock(
+      roundId,
+      facilitatorId2,
+      cb1
+    )
+
+    round ! SelectedUnionBlock(
+      roundId,
+      facilitatorId3,
+      cb2
+    )
+
+    round.underlyingActor.passToParentActor(StopBlockCreationRound(RoundId("round1"),None)) was called
+  }
+
+  test("it should return None when selectedCheckpointBlocks are empty") {
```

Comment:
it's the test for Round. imo there should be explicitly said what should return None because: `Round (it) should return None` could be misunderstood

Reply:
maybe: `it should finish round with no finished checkpoint block when selectedCheckpointBlocks are empty`
and above as well: `it should finish round with no finished checkpoint block when accept majority checkpoint block failed`